portable/IAR/AtmelSAM7S64/lib_AT91SAM7X256.h

```diff
@@ -1,1469 +1,1396 @@
+/** \fn    AT91F_PIO_ClearOutput */
+/** \brief Set to 0 output PIO */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_ClearOutput( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                         unsigned int flag ) /* \arg  output to be cleared */
+    {
+        pPio->PIO_CODR = flag;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_ForceOutput */
+/** \brief Force output when Direct drive option is enabled */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_ForceOutput( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                         unsigned int flag ) /* \arg  output to be forced */
+    {
+        pPio->PIO_ODSR = flag;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_Enable */
+/** \brief Enable PIO */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_Enable( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                    unsigned int flag ) /* \arg  pio to be enabled */
+    {
         pPio->PIO_PER = flag;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_Disable
-//* \brief Disable PIO
-//*----------------------------------------------------------------------------
-__inline void AT91F_PIO_Disable(
-        AT91PS_PIO pPio,   // \arg  pointer to a PIO controller
-        unsigned int flag) // \arg  pio to be disabled
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_Disable */
+/** \brief Disable PIO */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_Disable( AT91PS_PIO pPio,    /* \arg  pointer to a PIO controller */
+                                     unsigned int flag ) /* \arg  pio to be disabled */
+    {
         pPio->PIO_PDR = flag;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_PIO_GetStatus
-//* \brief Return PIO Status
-//*----------------------------------------------------------------------------
-__inline unsigned int AT91F_PIO_GetStatus( // \return PIO Status
-        AT91PS_PIO pPio) // \arg  pointer to a PIO controller
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_GetStatus */
+/** \brief Return PIO Status */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_PIO_GetStatus( /* \return PIO Status */
+        AT91PS_PIO pPio )                      /* \arg  pointer to a PIO controller */
```

Comment:
Should this be one line?

Reply:
Answered [here](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/768#discussion_r1313523527)